model/autosklearn/dffml_model_autosklearn/autoclassifier.py

```diff
@@ -0,0 +1,38 @@
+import importlib
+
+import autosklearn.classification
+
+from dffml.model.model import Model
+from dffml.util.entrypoint import entrypoint
+
+from .config import AutoSklearnConfig, AutoSklearnModelContext
+
+
+class AutoSklearnClassifierModelContext(AutoSklearnModelContext):
+    def __init__(self, parent):
+        super().__init__(parent)
+
+    @property
+    def model(self):
+        """
+        Generates or loads a model
+        """
+        if self._model is not None:
+            return self._model
+        self._model = autosklearn.classification.AutoSklearnClassifier(
```

Comment:
https://github.com/intel/dffml/blob/b1906cb37ed4305ebcc0a1f3c0957daa1234c784/model/scikit/dffml_model_scikit/scikit_base.py#L79-L83

Reply:
thanks for this :)